js/ui/text_box/ui.text_editor.base.js

```diff
@@ -442,6 +446,26 @@ const TextEditorBase = Editor.inherit({
         this._input().prop('spellcheck', this.option('spellcheck'));
     },
 
+    _cleanLabelObservable: function() {
+        if(this._$labelObservableTarget) {
+            resizeObserverSingleton.unobserve(this._$labelObservableTarget);
+        }
+    },
+
+    _setLabelWidth: function() {
+        this._$labelObservableTarget = $(this._$field ? this._$field : this._$tagsContainer ? this._$tagsContainer : this._input()).get(0);
+
+        const setWidth = () => {
+            const labelWidth = getWidth(this._$labelObservableTarget);
+            this._$label.find('.dx-label').css('maxWidth', labelWidth);
+        };
+
+        setWidth();
+
+        this._cleanLabelObservable();
```

Comment:
we already do it [here](https://github.com/DevExpress/DevExtreme/pull/19461/files#diff-d5f8ba33de2fd615f123442b1375acee5e5112c3357d545e514ca1838796c197R477)